app/maildelay/settings.py

```diff
@@ -202,10 +202,10 @@
 
 # Mailserver login settings
 
-EMAIL_HOST_USER = 'maildelay@maildelay.ml'
-DEFAULT_FROM_EMAIL = 'maildelay@maildelay.ml'
-EMAIL_HOST_PASSWORD = '3p7KDn4FugQQ'
-EMAIL_HOST = 'maildelay.ml'
+EMAIL_HOST_USER = 'maildelay@dev.rmd.io'
+DEFAULT_FROM_EMAIL = 'maildelay@dev.rmd.io'
+EMAIL_HOST_PASSWORD = 'KOhxgU8qHEtj'
+EMAIL_HOST = 'smtp.syhosting.ch'
 EMAIL_FOLDER = 'INBOX'
```

Comment:
Please don't put working credentials in the source, even if it's only for the dev env.

Change those credentials, and put some placeholder in here instead